### PR TITLE
Update Issue Template

### LINKED ISSUE #1144: Adding ISSUE_TEMPLATE.md file
There should be proper Issue Template for creating new issues so that others can know about that issue and its related thing.

### PR DESCRIPTION
**Pull request by ujjwalagrawal17**
## Description

Fixes #1144 

Added "Would you like to work on the issue section"

### CODE REVIEW

**Comment by ujjwalagrawal17** [comment]
@maskaravivek can we merge it ?

**Comment by maskaravivek** [comment]
Am waiting for someone else to review it as well. :)

File: ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -33,3 +33,7 @@ You can find this information by going to the navigation drawer in the app and t
 **Screen-shots:** 
 
 Can be created by pressing the Volume Down and Power Button at the same time on Android 4.0 and higher.
+
+**Would you like to work on the issue?**
+
+Please let us know if you can work on it or the issue should be assigned to someone else.
```
Comment (nicolas-raoul) [inline ISSUE_TEMPLATE.md]: These days many issues are submitted by developers, but most of the time issues are submitted by users who don't know anything about programming and might misunderstand this as "I can help by providing feedback". So I suggest changing to the following and then merging:

Please let us know whether you want to fix the issue by yourself. If not, anyone can get the issue assigned to them.

Comment (misaochan) [reply ISSUE_TEMPLATE.md]: Yeah, I agree with this. Happy to merge after this change is made. :)

Comment (ujjwalagrawal17) [reply ISSUE_TEMPLATE.md]: I had updated it.

**Pull request merged**